network/manifest-1/README.md

```diff
@@ -1,35 +1,44 @@
 # Mainnet Genesis
 
 TODO:
+
 - Update PoA Admin(s) from manifest10d07y265gmmuvt4z0w9aw880jnsr700jmq3jzm
 - Remove manifest1hj5fveer5cjtn4wd6wstzugjfdxzl0xp8ws9ct once others are given genesis allocations
```

Comment:
There are still TODOs?

Reply:
For mainnet, yes: Still need the multisig wallet address from the team so this can be completed. mainnet genesis can not be built without the address from the team. This address is a placeholder which we are using on the testnet.